python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -67,6 +69,19 @@ class FocalPlanePlot(PlotAction):
         doc="Operation to perform in binned_statistic_2d",
         default="mean",
     )
+    plotMin = Field[float](
+        doc="Minimum in z-value to display in the focal plane plot and in the histogram plot, if applicable",
+        default=None,
+        optional=True,
+    )
+    plotMax = Field[float](
+        doc="Maximum in z-value to display in the focal plane plot and in the histogram plot, if applicable",
+        default=None,
+        optional=True,
+    )
+    showStats = Field[bool](doc="Show statistics for plotted data", default=True)
+    addHistogram = Field[bool](doc="Add a histogram of all input points", default=False)
+    histBins = Field[int](doc="Number of bins to use in histogram", default=30)
```

Comment:
Consider using `HistPanel` from histPlot rather than having parallel implementations.

Reply:
If I just use `HistPanel` for the configuration, there are a lot of additional config options I would need to implement. Or I could fully take advantage of the `HistPlot` class and make an instance of `HistPlot`, then use `HistPlot._make_panel()`, but obviously this is private. Do you have something else in mind?

Reply:
I haven't looked at the generic multi-panel plot ticket yet, but in the meantime I think you can add a config field like `hist = ConfigurableActionField[HistPlot]`.

Looking at `HistPanel` a bit more closely, I think `HistPlot._make_panel` should essentially move wholesale to `HistConfig.make_panel`; the only place `self` is needed is to refer to `self.panels[panel]` and to other single-underscore methods that similarly could be moved to `HistConfig`. But that is a non-trivial refactor that probably should go on another ticket.

Reply:
I don't have an issue with calling `_make_panel` - that should be recommended usage (but like I said, it's not a trivial fix, it needs to be split off from `HistPlot` and renamed `make_panel`).